spec/services/idv/profile_maker_spec.rb

```diff
@@ -59,6 +61,7 @@
           fraud_review_needed: true,
           gpo_verification_needed: false,
           deactivation_reason: nil,
+          fraud_provider_status: nil,
```

Comment:
I think `fraud_provider_status` should either be `threatmetrix_review` or `threatmetrix_reject` here for consistency with `fraud_review_needed: true`